pyanaconda/exception.py

```diff
@@ -190,9 +198,14 @@ def handleException(self, dump_info):
                 # data and let message handler run the exception handler in
                 # the main thread
                 exc_info = dump_info.exc_info
-                hubQ.send_exception((exc_info.type,
-                                     exc_info.value,
-                                     exc_info.stack))
+                if App.is_initialized():
```

Comment:
I think it might make sense to document around here that this is basically a split between handling exception via TUI and GUI.

Also I wonder - what about the ask-vnc-question TUI screen ? Could that result in Simpleline App being initialized while the the user is actually using the GUI over VNC ?

Reply:
I've added comment to explain why this `if` is here. If I'm not mistaken the `ask-vnc-question` screen is not using python-meh and if it is using python-meh, then the Simpleline would be initialized so or so.